ember-cli-build.js

```diff
@@ -3,6 +3,11 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 module.exports = function (defaults) {
   const app = new EmberApp(defaults, {
+    babel: { sourceMaps: 'inline' },
```

Comment:
Ik begreep van je bericht in chat dat deze instellingen nodig zijn om het debuggen in je editor te vereenvoudigen. Het is erg goed dat je dat uitgezocht hebt, want dit soort dingen maken het development-proces efficiënte, vaak meer dan we denken. Nu is het wel zo dat deze configuratie committen, invloed zou hebben op ieders setup. Zeker voor de gehoste versies is het bv niet gewenst om sourceMaps inline te includen. Gebruik deze aanpassingen dus voorlopig best lokaal (graag deze aanpassing terug uit de PR verwijderen). We zullen later bekijken hoe we dit kunnen configureren om zowel lokaal als op de server de juiste configuratie te hebben.

Reply:
OK, had ik beter gevraagd